src/wrapper/SerializationWrapper.cpp

```diff
@@ -67,6 +67,70 @@ char fs_serializer_openmode(void* serializer)
     return ' ';
 }
 
+int fs_serializer_metainfo(void* serializer)
+{
+    return reinterpret_cast<const Serializer*>(serializer)->globalMetainfo().size();
+}
+
+void fs_serializer_metainfo_key_lengths(void* serializer, int* lengths)
```

Comment:
Could we assert null pointers?
Also I was thinking that it is easy to go out of bounds here, if the user does not allocate enough list for lengths in python?
Any way of protecting a bit this code against bad usage? How about using boost python?
Could we interoperate between std vector and boost::python::list for lengths?
http://stackoverflow.com/questions/6157409/stdvector-to-boostpythonlist